client/platform/web-girder/views/DataShared.vue

```diff
@@ -0,0 +1,116 @@
+
+
+export default defineComponent({
+  name: 'DataShared',
+  setup() {
+    const total = ref();
+    const dataList = ref([] as GirderModel[]);
+    const tableOptions = reactive({
+      page: 1,
+      itemsPerPage: 10,
+      sortBy: ['created'],
+      sortDesc: [true],
+    } as DataOptions);
+    const store = useStore();
+    const { getters } = store;
+    const locationStore = store.state.Location;
+
+    const headers = [
+      { text: 'File Name', value: 'name' },
+      { text: '', value: 'annotator', sortable: false },
+      { text: 'File Size', value: 'formattedSize' },
+      { text: 'Shared By', value: 'ownerLogin' },
+    ];
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const fixSize: any = mixins.sizeFormatter.methods;
+
+    const updateOptions = async () => {
+      const {
+        sortBy, page, itemsPerPage, sortDesc,
+      } = tableOptions;
+      const limit = itemsPerPage;
+      const offset = (page - 1) * itemsPerPage;
+      const sort = sortBy[0] || 'created';
+      const sortDir = sortDesc[0] === false ? 1 : -1;
+      const shared = true;
+
+      const response = await getDatasetList(limit, offset, sort, sortDir, shared);
+      dataList.value = response.data;
+      total.value = Number.parseInt(response.headers['girder-total-count'], 10);
+      dataList.value.forEach((element) => {
+        // eslint-disable-next-line no-param-reassign
+        element.formattedSize = fixSize.formatSize(element.size);
+      });
+    };
+
+    function setLocation(location: LocationType) {
+      store.dispatch('Location/setRouteFromLocation', location);
+    }
+
+    function isAnnotationFolder(item: GirderModel) {
+      return item._modelType === 'folder' && item.meta.annotate;
+    }
+
+    watch(tableOptions, updateOptions, {
+      deep: true,
+    });
+
+    updateOptions();
+    return {
+      isAnnotationFolder,
+      dataList,
+      getters,
+      updateOptions,
+      setLocation,
+      total,
+      locationStore,
+      ...toRefs(tableOptions),
+      headers,
+    };
+  },
+});
+
+</script>
+
+<template>
+  <v-data-table
+    v-model="locationStore.selected"
+    :selectable="!getters['Location/locationIsViameFolder']"
+    :location="locationStore.location"
+    :headers="headers"
+    :page.sync="page"
+    :items-per-page.sync="itemsPerPage"
+    :sort-by.sync="sortBy"
+    :sort-desc.sync="sortDesc"
+    :server-items-length="total"
+    :items="dataList"
+    item-key="_id"
+    show-select
+    @input="$emit('input', $event)"
+    @update:location="setLocation"
+  >
+    <!-- eslint-disable-next-line -->
+    <template v-slot:item.annotator="{item}">
+      <v-btn
+        v-if="isAnnotationFolder(item)"
+        class="ml-2"
+        x-small
+        color="primary"
+        depressed
+        :to="{ name: 'viewer', params: { id: item._id } }"
+        @click.stop="openClip(item)"
+      >
+        Launch Annotator
+      </v-btn>
+    </template>
+  </v-data-table>
```

Comment:
``` html
    <template #no-data>
      <span class="pr-4">No datasets have been shared with you yet.</span>
      <a href="https://kitware.github.io/dive/Web-Version/#sharing-data-with-teams">Learn more about sharing</a>
    </template>
```
I support aggressively linking to documentation.  Can we add this slot here?

![Screenshot from 2021-09-21 13-53-15](https://user-images.githubusercontent.com/4214172/134222396-b787f993-ad9d-408d-9e52-3766859f4532.png)

Reply:
Definitely. I think this is a great idea.